boards/arm/teensy4/dts_fixup.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2020 Bernhard Kraemer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Needed for CODE and DATA relocation */
+#define ITCM_ADDR DT_N_S_soc_S_flexram_400b0000_S_itcm_0_REG_IDX_0_VAL_ADDRESS
+#define DTCM_ADDR DT_N_S_soc_S_flexram_400b0000_S_dtcm_20000000_REG_IDX_0_VAL_ADDRESS
+#define OCRAM_ADDR DT_N_S_soc_S_flexram_400b0000_S_ocram_20200000_REG_IDX_0_VAL_ADDRESS
```

Comment:
We're trying to avoid dts fixups. Do you still need this after commit c6cc927506fddc50445c767460b486f91b038578?